js/index.js

```diff
@@ -0,0 +1,59 @@
+$(function(){
+ 
+// listen for user clicking 'submit'/submitting text
+
+$("#submit-btn").click(changeBackground);
+// $("#submit-btn").click(convertTemp);
+
+function convertTemp() {
+ // Set the initial variables for c (Celsius) and f (Fahrenheit)
+ var ctemp = document.getElementById('ctemp'), ftemp = document.getElementById('ftemp');
```

Comment:
@jcooper8190 if you chose to use one var and comma delimitate the rest of the variables. Try to use this convention. 

'var ctemp = document.getElementById('ctemp'),
       ftemp = document.getElementById('ftemp');
`